nncf/common/graph/graph.py

```diff
@@ -178,6 +178,19 @@ def get_previous_nodes(self, node: NNCFNode) -> List[NNCFNode]:
         nx_node_keys = self._nx_graph.pred[self._node_id_to_key_dict[node.node_id]]
         return [self._nx_node_to_nncf_node(self._nx_graph.nodes[key]) for key in nx_node_keys]
 
+    def get_previous_nodes_sorted_by_in_port(self, node: NNCFNode) -> List[NNCFNode]:
+        """
+        Returns producer nodes of provided node sorted by 'in_port'.
+
+        :param node: Consumer node.
+        :return: List of producers nodes of provided node.
```

Comment:
`:return: List of producers nodes of provided node.` -> `:return: List of producers nodes of provided node sorted by in_port.`